packages/core/src/api/interactions.ts

```diff
@@ -71,14 +82,16 @@ export class InteractionsAPI {
 	 * @see {@link https://discord.com/developers/docs/interactions/receiving-and-responding#create-followup-message}
 	 * @param applicationId - The application id of the interaction
 	 * @param interactionToken - The token of the interaction
-	 * @param data - The data to use when replying
+	 * @param body - The data to use when replying
+	 * @param options - The options to use when replying
 	 */
 	public async followUp(
 		applicationId: Snowflake,
 		interactionToken: string,
-		data: APIInteractionResponseCallbackData & { files?: RawFile[] },
+		body: APIInteractionResponseCallbackData & { files?: RawFile[] },
+		options: Pick<RequestData, 'signal'> = {},
```

Comment:
`options:` fields don't need defaulting to an empty object if they're not accessed.